admin/js/novaramedia-audio-tool-admin.js

```diff
@@ -120,6 +125,75 @@
 
       $('#output-archive-org-link').attr('href', encodeURI(archiveUrl));
 
+      // draw artwork
+
+      _this.drawArtwork(data.post_image);
```

Comment:
Posts without images should be handled here cuz `data.post_image === false` when they don't have an image.